stance/main.py

```diff
@@ -30,20 +32,33 @@
         with open("squatterrible_user_skeletons.json", 'rb') as fp:
             user_skeletons = pickle.load(fp)
     else:
+        logger.error("Error parsing command line arguments")
         raise ValueError("The pickle for the motion video {} given has not been computed,"
                          " please run without --use_pickle".format(args.motion_video))
 
     # Find when the user was in a benchmark zone
-    print("Finding benchmark zones")
+    logger.info("Finding Benchmark Zones")
```

Comment:
For the logs in main, can you do something like "Found benchmark zones in 'some-time-in-seconds' seconds"? I think it would make the logs have more relevant info for us